### PR TITLE
Add a new "button" item type to the Form

### PR DESCRIPTION
**Pull request by DokaRus**

### CODE REVIEW

File: js/docEnums.js [hunk 1]
```diff
@@ -384,6 +384,11 @@
  * @enum {'empty'|'group'|'simple'|'tabbed'}
  */
 
+ /**
+ * @typedef {string} Enums.FormItemAlignment
```
Comment (DokaRus) [inline js/docEnums.js]: @San4es Is it a good name for an MVC enum?

**Review by TikhonovaTatyana — DISMISSED** [review]

File: js/docEnums.js (hunk 1 above)
Comment (San4es) [reply js/docEnums.js]: LGTM

File: js/docEnums.js [hunk 2]
```diff
@@ -384,6 +384,11 @@
...
+ * @enum {'center'|'left'|'right'|'start'|'end'}
```
Comment (San4es) [inline js/docEnums.js]: I suggest using an alphabetical order for enum items.

File: js/ui/form/ui.form.js [hunk 3]
```diff
@@ -635,6 +635,60 @@ var Form = Widget.inherit({
...
+            /**
+             * @name dxFormButtonItem_colSpan
+             * @publicName colSpan
+             * @type number
+             * @default undefined
+             */
+            /**
+             * @name dxFormButtonItem_itemType
+             * @publicName itemType
+             * @type Enums.FormItemType
```
Comment (San4es) [inline js/ui/form/ui.form.js:659]: Do we need to add the "button" item to the "Enums.FormItemType" enum?

**Review by San4es — CHANGES_REQUESTED** [review]

File: js/ui/form/ui.form.js (hunk 3 above)
Comment (DokaRus) [reply js/ui/form/ui.form.js:659]: Yes, I'll fix it

**Review by San4es — DISMISSED** [review]

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**